events/metrics.go

```diff
@@ -0,0 +1,30 @@
+package events
+
+import (
+	pmetrics "github.com/awslabs/operatorpkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	MetricLabelEventType   = "type"
+	MetricLabelEventReason = "reason"
```

Comment:
```suggestion
	MetricLabelType   = "type"
	MetricLabelReason = "reason"
```
Pretty sure it be just this because it's scoped to the `events` pkg already, right?

Reply:
Here I was following the convention we had set out in the status condition controller https://github.com/awslabs/operatorpkg/blob/main/status/metrics.go#L14-L16.

Reply:
I can go a head and make this common if that's the approach we want to take